functions/openhabian.bash

```diff
@@ -440,3 +440,32 @@ create_user_and_group() {
     cond_redirect usermod --append --groups openhab,sudo "$userName" &> /dev/null
   fi
 }
+
+## import initial configuration
+##
+##    import_openhab_config()
+##
+## valid argument: file name to import using openhab-cli
+##
+import_openhab_config() {
+  local initialConfig=${initialconfig:-/boot/initial.zip}
+  local restoreFile=${initialConfig}
+
+  echo -n "$(timestamp) [openHABian] Importing initial openHAB configuration... "
+  if [[ $# -eq 1 ]]; then
+    initialConfig=$1
+  fi
+  if [[ "$initialConfig" =~ http:* ]]; then
+    restoreFile="$(mktemp "${TMPDIR:-/tmp}"/openhabian.XXXXX)"
+    cond_redirect wget -qO "$restoreFile" "$initialConfig"
+  fi
+
+  if [[ -n $UNATTENDED ]] && [[ ! -f $initialConfig ]]; then
```

Comment:
Might as well leave the SKIPPED message as 'no config file provided'.

Reply:
I prefer my version as it gives an indication on filename typos

Reply:
Thats a pretty good reason, I'm OK with it